content/projects/metabake.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: Metabake
-repo: metabake/mbake-CLI-for-Metabake
-homepage: http://www.metabake.net
+repo: metabake/MetaBake-Examples
```

Comment:
The repo should be the repo of the static site generator itself.

Reply:
OK, fixed. I assume I added corrected code to the PR correctly.